lib/msal-browser/docs/caching.md

```diff
@@ -1,3 +1,88 @@
-## Cache Storage
+# Caching
 
-We offer two methods of storage for Msal, `localStorage` and `sessionStorage`.  Our recommendation is to use `sessionStorage` because it is more secure in storing tokens that are acquired by your users, but `localStorage` will give you Single Sign On accross tabs and user sessions.  We encourage you to explore the options and make the best decision for your application.
+When MSAL acquires a token, it caches it for future usage. MSAL manages token lifetimes and refreshing for you. APIs like `acquireTokenSilent()` retrieves tokens from the cache for a given account.
```

Comment:
Can we list all the APIs that retrieve cached tokens?

Reply:
Just added. Would that work?